test/resource_cluster_test.go

```diff
@@ -354,56 +354,56 @@ func TestValidRedisClusterCreate(t *testing.T) {
 	})
 }
 
-func TestAccClusterCredentials(t *testing.T) {
-	testAccProviders := map[string]terraform.ResourceProvider{
-		"instaclustr": instaclustr.Provider(),
-	}
-	validConfig, _ := ioutil.ReadFile("data/valid_with_password_and_client_encryption.tf")
-	username := os.Getenv("IC_USERNAME")
-	apiKey := os.Getenv("IC_API_KEY")
-	hostname := getOptionalEnv("IC_API_URL", instaclustr.DefaultApiHostname)
-	oriConfig := fmt.Sprintf(string(validConfig), username, apiKey, hostname)
+//func TestAccClusterCredentials(t *testing.T) {
```

Comment:
**[Required]** We shouldn't be commenting out tests, does this pass with the changes made in this PR?

Reply:
fixed as opportunistic